src/test/java/com/suse/salt/netapi/calls/modules/TimezoneTest.java

```diff
@@ -0,0 +1,85 @@
+package com.suse.salt.netapi.calls.modules;
+
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import com.google.gson.JsonNull;
+import com.suse.salt.netapi.calls.LocalCall;
+import com.suse.salt.netapi.client.SaltClient;
+import com.suse.salt.netapi.client.impl.HttpAsyncClientImpl;
+import com.suse.salt.netapi.datatypes.AuthMethod;
+import com.suse.salt.netapi.datatypes.Token;
+import com.suse.salt.netapi.datatypes.target.MinionList;
+import com.suse.salt.netapi.errors.JsonParsingError;
+import com.suse.salt.netapi.results.Result;
+import com.suse.salt.netapi.utils.ClientUtils;
+import com.suse.salt.netapi.utils.TestUtils;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.util.Map;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.any;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlMatching;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Timezone module unit tests.
+ */
+public class TimezoneTest {
+
+    private static final int MOCK_HTTP_PORT = 8888;
+
+    private static final String JSON_NULL_RESPONSE = ClientUtils.streamToString(
+            TimezoneTest.class.getResourceAsStream(
+            "/modules/timezone/null_response.json"));
+
+    private static final String JSON_GETOFFESET_OK_RESPONSE = ClientUtils.streamToString(
+            TimezoneTest.class.getResourceAsStream(
+            "/modules/timezone/getOffset_ok.json"));
```

Comment:
I might suggest to rename that file in order to not mix underscore and camel case notations to `get_offset_ok.json`.